config/config.go

```diff
@@ -1,14 +1,15 @@
-package aibridge
+package config
 
-type ProviderConfig struct {
-	BaseURL, Key string
-}
-
-type (
-	OpenAIConfig    ProviderConfig
-	AnthropicConfig ProviderConfig
+const (
+	ProviderAnthropic = "anthropic"
+	ProviderOpenAI    = "openai"
 )
 
+type AnthropicConfig struct {
```

Comment:
Nit: since this is now in the `config` package, these structs no longer need the `Config` suffix.